src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -40,30 +40,34 @@ string GetVisualStudio2017Directory ()
 
 		public string XABuildExe {
 			get {
+				string xabuild;
 				if (IsUnix) {
 					RunningMSBuild = true;
 					var useMSBuild = Environment.GetEnvironmentVariable ("USE_MSBUILD");
 					if (!string.IsNullOrEmpty (useMSBuild) && useMSBuild == "0") {
 						RunningMSBuild = false;
 					}
 					#if DEBUG
-					var xabuild = Path.GetFullPath (Path.Combine (Root, "..", "Debug", "bin", "xabuild"));
+					xabuild = Path.GetFullPath (Path.Combine (Root, "..", "Debug", "bin", "xabuild"));
 					#else
-					var xabuild = Path.GetFullPath (Path.Combine (Root, "..", "Release", "bin", "xabuild"));
+					xabuild = Path.GetFullPath (Path.Combine (Root, "..", "Release", "bin", "xabuild"));
 					#endif
 					if (File.Exists (xabuild))
 						return xabuild;
 					xabuild = Path.GetFullPath (Path.Combine (Root, "..", "..", "..", "..", "..", "..", "..", "out", "bin", "xabuild"));
 					if (File.Exists (xabuild))
 						return xabuild;
-					return Path.GetFullPath (Path.Combine (Root, "..", "..", "tools", "scripts", "xabuild"));
+					return RunningMSBuild ? "msbuild" : "xbuild";
 				}
 
 				#if DEBUG
-				return Path.GetFullPath (Path.Combine (Root, "..", "..", "bin", "Debug", "bin", "xabuild.exe"));
+				xabuild = Path.GetFullPath (Path.Combine (Root, "..", "..", "bin", "Debug", "bin", "xabuild.exe"));
 				#else
-				return Path.GetFullPath (Path.Combine (Root, "..", "..", "bin", "Release", "bin", "xabuild.exe"));
```

Comment:
I think I wrote this, but we could simplify the paths on Windows to match what Mac has.

Instead of:
```
return Path.GetFullPath (Path.Combine (Root, "..", "..", "bin", "Debug", "bin", "xabuild.exe"));
```

Do something like:
```
return Path.GetFullPath (Path.Combine (Root, "..", "Debug", "bin", "xabuild.exe"));
```